### ISSUE TITLE
Upgrading from pre 9.5 instructions are not clear

### ISSUE THREAD

**Issue by axelson** [PostgresApp/PostgresApp#399]
I mentioned this on #388 but I wanted to create a separate issue to hopefully get the upgrade instructions tweaked.

The https://postgresapp.com/documentation/update.html and https://postgresapp.com/documentation/migrating-data.html pages are missing a crucial step if you are trying to upgrade from something a pre 9.5 database.

So PostgresApp 2 has this (great!) concept of multiple databases, and it ships with the **binaries** for 9.5 and 9.6, so if you are currently running 9.4 and you upgrade to PostgresApp 2 you will get this "Binaries not found" error. However, you are not getting this error for the entire PostgresApp 2 application (although that's easy to assume for a new user of Postgres 2, speaking from personal experience), just for your existing 9.4 server instance. So if you click on the panel icon in the bottom left:
<img width="94" alt="screenshot 2017-07-19 15 51 14" src="https://user-images.githubusercontent.com/9973/28396918-1ef95a70-6c9a-11e7-9788-082894f92e50.png">

You can create a new server running the latest version of postgres and then you can finish following the instructions on https://postgresapp.com/documentation/migrating-data.html

I think this info would be good to add to either https://postgresapp.com/documentation/update.html and https://postgresapp.com/documentation/migrating-data.html

**Comment by jakob** [comment]
Thanks for your comments. You are absolutely right that this is a problem. It isn't obvious that Postgres.app supports multiple servers with different versions.

However, I don't think we should attempt to fix this by changing the documentation -- I think we should change Postgres.app to make it clear what is going on, without requiring users to read the docs. (most people read docs as a last resort, when all else fails, and they are already frustrated by the crappy software)

So here are my thoughts on this:

1. Most users don't care about installing multiple versions. They just want a single PostgreSQL server running the most recent version. This is what Postgres.app offers when you first start it. We hide the sidebar so that this type of user doesn't need to think about installing multiple servers. It looks like there is only a single server, which is exactly what 99% of users want.

2. Users who already have an old Postgres.app installation on their Mac will be confused by this. As you explained, it looks like everything is broken. And even if the version is supported (eg. 9.5), it's not obvious how to upgrade to 9.6

To deal with the second type of user, I suggest the following changes

- show the sidebar by default
- if the user does not have a server of the latest version, create a second server with the latest version

This would allow users to immediately use the new server (if they want) while also making it obvious that the old data isn't lost. And the docs explain how to migrate the data.

**Comment by axelson** [comment]
That sounds like a reasonable solution to me. Although my preference would be a doc update in addition to the application update.

**Comment by jakob** [comment]
Yes, updating the docs in additionwould be great – I’m not sure if there is any mention of the multi-server capabilities in the current docs.

However, I’m not sure where to put that info, and what exactly to write.

The migration docs are currently very generic – if you have any suggestions how to improve them, I’d welcome them!

**Comment by saschafrohwerk** [comment]
I woud also add that the documentation of the data migration could be clearer. When you just enter the commands as provided you get "command not found". So we should add the paths (eg from 9.5 to 10 ):

/Applications/Postgres.app/Contents/Versions/9.5/bin/pg_dumpall --quote-all-identifiers | gzip >postgresapp.sql.gz

and

gunzip <postgresapp.sql.gz | /Applications/Postgres.app/Contents/Versions/10/bin/psql

**Comment by gs2589** [comment]
On this note:  I had the older version of Postgres app running 9.4. 
I was able to run things like postgres -V and get a version.

I deleted the old app and installed the new one. Now postgres -V doesn't work. I get:
-bash: /Applications/Postgres.app/Contents/Versions/9.4/bin/postgres: No such file or directory

Could someone please help me figure out what I need to change so that my terminal commands now point to the new installation.

**Comment by gs2589** [comment]
Fixed it by running:
sudo mkdir -p /etc/paths.d && echo /Applications/Postgres.app/Contents/Versions/latest/bin | sudo tee /etc/paths.d/postgresa

Important to close and reopen window.

**Comment by unformatt** [comment]
> if the user does not have a server of the latest version, create a second server with the latest version

This doesn't seem to be happening. It would be nice if the UI was like "You have no binaries installed. What to download the latest?"

![image](https://user-images.githubusercontent.com/157132/62383702-5dde3200-b51e-11e9-98ab-491b6bf72a9f.png)
